src/uu/cksum/src/cksum.rs

```diff
@@ -148,4 +202,12 @@ pub fn uu_app() -> Command {
                 .action(clap::ArgAction::Append)
                 .value_hint(clap::ValueHint::FilePath),
         )
+        .arg(
+            Arg::new(options::ALGORITHM)
+                .long(options::ALGORITHM)
+                .short('a')
+                .help("select the digest type to use. See DIGEST below")
+                .value_name("ALGORITHM"),
```

Comment:
Here you can add `.value_parser(["bsd", "sysv", ...])` to actually restrict the arguments. Tha't's what I was referring to in the other comment. Here's an example of `du` doing the same thing: https://github.com/uutils/coreutils/blob/main/src/uu/du/src/du.rs#L877.

Reply:
Thanks for advice. I have put a fix